pkg/authorization/apis/authorization/conversion.go

```diff
@@ -241,3 +243,42 @@ func Convert_rbac_PolicyRules_To_authorization_PolicyRules(in []rbac.PolicyRule)
 	}
 	return rules
 }
+
+func copyMapExcept(in map[string]string, except string) map[string]string {
+	out := map[string]string{}
+	for k, v := range in {
+		if k != except {
+			out[k] = v
+		}
+	}
+	return out
+}
+
+func convert_Origin_Annotations_To_rbac_Annotations(in map[string]string) map[string]string {
+	// can't use const in pkg/oc/admin/policy because of import cycle
+	value, ok := in["openshift.io/reconcile-protect"]
+	if ok {
+		out := copyMapExcept(in, "openshift.io/reconcile-protect")
+		if value == "true" {
+			out[rbac.AutoUpdateAnnotationKey] = "false"
+		} else {
```

Comment:
> So unless there is an actual detrimental effect in preserving explict annotations (whatever the value) I feel it is safer to keep them even if the value they represent is the default action currently taken by the bootstrap reconciliation code.

1. "true" (protected) -> "false" (protected)
2. "false" (unprotected) -> "true" (unprotected)
3. "other" (unprotected?) -> I'd keep the original map key and original map value.  That preserves both round-tripping and protection status, right?